recipes/_install-fips-proxy-linux.rb

```
@@ -0,0 +1,76 @@
+
+dd_fips_proxy_version = Chef::Datadog.fips_proxy_version(node)
+dd_fips_proxy_package_name = node['datadog']['fips_proxy_package_name']
+
+package_action = node['datadog']['fips_proxy_package_action']
+
+package_retries = node['datadog']['fips_proxy_package_retries']
+package_retry_delay = node['datadog']['fips_proxy_package_retry_delay']
+
+# Install the regular package
+case node['platform_family']
+when 'debian'
+  apt_package dd_fips_proxy_package_name do
+    version dd_fips_proxy_version
+    retries package_retries unless package_retries.nil?
+    retry_delay package_retry_delay unless package_retry_delay.nil?
+    action package_action # default is :install
+    options '--force-yes' if node['datadog']['fips_proxy_allow_downgrade']
+  end
+
+  apt_package 'datadog-signing-keys' do
+    retries package_retries unless package_retries.nil?
+    retry_delay package_retry_delay unless package_retry_delay.nil?
+    action :upgrade
+  end
+when 'rhel', 'fedora', 'amazon'
+  if platform_family?('rhel') && node['platform_version'].to_i >= 8 && !platform?('amazon') ||
+     platform_family?('fedora') && node['platform_version'].to_i >= 28
+    # yum_package doesn't work on RHEL 8 and Fedora >= 28
+    # dnf_package only works on RHEL 8 / Fedora >= 28 if Chef 15+ is used
+    dnf_package dd_fips_proxy_package_name do
+      version dd_fips_proxy_version
+      retries package_retries unless package_retries.nil?
+      retry_delay package_retry_delay unless package_retry_delay.nil?
+      action package_action # default is :install
+    end
+  else
+    yum_package dd_fips_proxy_package_name do
+      version dd_fips_proxy_version
+      retries package_retries unless package_retries.nil?
+      retry_delay package_retry_delay unless package_retry_delay.nil?
+      action package_action # default is :install
+      allow_downgrade node['datadog']['fips_proxy_allow_downgrade']
+    end
+  end
+when 'suse'
```

I think we don't have SUSE packages for fips-proxy, meaning that this part should be excluded and we should fail early for SUSE in the dd-fips-proxy recipe (similar to what we do for Windows)